app/views/devise/sessions/new.html.erb

```diff
@@ -1,37 +1,31 @@
-<% content_for :title, "Sign in" %>
-<% content_for :thin_form, true %>
+<% content_for :title, "Sign in to GOV.UK" %>
 
-<div class="page-title">
-  <h1>Sign in</h1>
-</div>
-
-<!--[if lte IE 7]>
```

Comment:
Can we still keep this alert wrapped in a conditional IE7 tag?

Reply:
IRL conversation: let's move this up to the layout or components gem.